sycl/include/CL/sycl/INTEL/esimd/detail/esimd_intrin.hpp

```diff
@@ -18,7 +18,8 @@
 #include <assert.h>
 #include <cstdint>
 
-#define __SIGD sycl::INTEL::gpu::detail
+#define __SEIEED sycl::ext::intel::experimental::esimd::detail
```

Comment:
For a future change, I'd like to see these macro names not be first letter of each of the namespaces.  I'd like to have you choose a name like __ESIMD_NS that was at least a bit meaningful.  But more importantly when ESIMD would move out of experimental, or otherwise change, I don't think you should have to change every single place these macros are used to designate the namespace.  The changes to these added to a huge number of extra changes in this PR.

Reply:
Good point. Maybe it is better to do this change in this PR and retest to avoid disturbance in future. @pvchupin - what do you think?

Reply:
@kbsmith-intel, how about:
__ESIMD_NS, __ESIMD_DETAIL_NS, __ESIMD_EMU_NS, __ESIMD_EMU_DETAIL_NS
?

Reply:
Those names look good to me.

Reply:
These are internal macro, right? I'd prefer it to go as a follow up change. Since we already committed tests we should commit the change asap.

Reply:
Yes, those are internal. OK, I will make this as a follow-up PR

Reply:
I think followup would be fine also, and agree with Pavel that since tests have changed, getting source changes in ASAP is desired.